apps/payment/app_internal_test.go

```diff
@@ -38,9 +38,9 @@ func TestApp_ValidInit(t *testing.T) {
 	app := new(App)
 
 	nildata := &channel.State{Data: nil}
-	assert.Panics(func() { app.ValidInit(nil, nildata) })
+	assert.Panics(func() { _ = app.ValidInit(nil, nildata) })
```

Comment:
We talked about the use of `_ =` vs `// nolint` on monday with the example of  
`defer func() { _ = closer.Closer() } ()`  
as opposed to
`defer closer.Closer() // nolint: errcheck`  
and came to the conclusion that suppressing was worse.  
Which is why i now always used `_ = ` instead of suppressing the linter.

Reply:
We talked about it and that was not my conclusion. You may have gotten me wrong. Next time ask if you are unsure and I will try to be more clear.

Reply:
Okay anyway, i updated it.  
@matthiasgeihs